apps/desktop/src/platform/services/ephemeral-value-storage.main.service.ts

```diff
@@ -0,0 +1,21 @@
+import { ipcMain } from "electron";
+
+/**
+ * The ephemeral value store holds values that should be accessible to the renderer past a process reload.
+ * In the current state, no keys, that alone can be used to decrypt a vault should be kept in this store.
```

Comment:
Small nit: I had some trouble parsing this line, and think it could be improved a little bit.
```suggestion
 * In the current state, this store must not contain any keys that can decrypt a vault by themselves.
```